Packs/OTRS/Integrations/OTRS/OTRS.py

```diff
@@ -558,10 +551,36 @@ def fetch_incidents():
     demisto.setLastRun({'time': last_created})
 
 
+def update_session():
+    client.session_create()
+    sessionID = client.session_id_store.value
+    demisto.setIntegrationContext({'SessionID': sessionID})
+    client.session_id_store.write(sessionID)
+
+
+def execute_otrs_method(method_name, args):
+    method = getattr(client, method_name)
```

Comment:
you can also have it as a decorator instead
```suggestion
def execute_otrs_method(method, args):
```